src/Administration/Resources/app/administration/src/module/sw-newsletter-recipient/page/sw-newsletter-recipient-list/index.js

```diff
@@ -59,11 +59,11 @@ Component.register('sw-newsletter-recipient-list', {
             this.tagCollection = new EntityCollection('/tag', 'tag', Shopware.Context.api, new Criteria());
 
             const criteria = new Criteria(1, 100);
```

Comment:
just a JS-noob question :grin: 

wouldn't it now makes sense to remove such occurrences of setting the limit to 100/500? they are just workarrounds, aren't they? 

or is there a benefit to leave them?

Reply:
It really is just to set the page size so we make bigger steps. When we start to refactor these calls for the page size we might also refactor all of them to keep https://github.com/shopware/platform/pull/2070 in mind. Not sure whether we should pull on all fuzzy threads :)

Reply:
okay :+1: thanks for clearing this up